jaggery-modules/rxt/module/scripts/asset/asset.js

```diff
@@ -1633,6 +1633,38 @@ var asset = {};
         }
         return modAsset;
     };
+    /**
+     * This method is used to format a timestamp to 'd:h:m:s'.
+     *
+     * @param duration  timestamp duration in milliseconds.
+     * @return String   formatted time duration to 'd:h:m:s'.
+     */
+    AssetManager.prototype.formatLCStateTimeDuration = function (duration) {
+        var mil = duration;
+        var timeDuration;
+        var seconds = (mil / 1000) | 0;
```

Comment:
Thank you Sameera,
This entire code block has been refactored with new time in lifecycle feature.